syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -588,6 +588,11 @@ def fix_prec_(self, *args, **kwargs):
 
     fix_precision_ = fix_prec_
 
+    def paillier(self, *args, **kwargs): # adds paillier encryption functionality to all 
```

Comment:
One question this change raises is what tensor types should have convenience methods in native.py for their creation. Theoretically going forward the api for native tensors gets incredibly cluttered if we add these connivence methods for every tensor type